finality-aleph/src/sync/forest/vertex.rs

```diff
@@ -50,6 +50,22 @@ impl<I: PeerId, J: Justification> Vertex<I, J> {
         }
     }
 
+    /// Whether the vertex is auxiliary.
+    pub fn auxiliary(&self) -> bool {
```

Comment:
```suggestion
    pub fn is_auxiliary(&self) -> bool {
```

Reply:
this is named analogically to other methods here, e.g. `importable`, `requestable`, `imported` (@timorl likes this convention)

Reply:
hmmm